bookkeeper-server/src/test/java/org/apache/bookkeeper/test/BookieZKExpireTest.java

```diff
@@ -67,7 +68,7 @@ public void testBookieServerZKRequestTimeoutBehaviour() throws Exception {
             Thread[] threads = new Thread[threadCount * 2];
             threadCount = Thread.enumerate(threads);
             for (int i = 0; i < threadCount; i++) {
-                if (threads[i].getName().indexOf("SendThread") != -1) {
+                if (threads[i].getName().contains("SendThread")) {
```

Comment:
nit: this change doesn't seem relevant

Reply:
@eolivelli Thanks for your review. I have reverted this minor change.